developer-docs-site/docs/tutorials/index.md

```diff
@@ -7,7 +7,9 @@ slug: "aptos-quickstarts"
 
 If you are new to the Aptos blockchain, begin with these quickstarts before you get into in-depth development. These tutorials will help you become familiar with how to develop for the Aptos blockchain using the Aptos SDK.
 
-### Install macOS prerequisites
+Note, they should be used only in the [Aptos devnet or testnet environments](../nodes/aptos-deployments.md).
```

Comment:
I forgot why we added just Mac here... but I really don't like it :/

It feels either out of place as it is mac only or insensitive as it is an overpriced development platform.

Reply:
we could probably make this generic or have tabs... I don't think there's much here that is mac specific except the brew commands

Reply:
> I forgot why we added just Mac here... but I really don't like it :/
> 
> It feels either out of place as it is mac only or insensitive as it is an overpriced development platform.

FWIW, this used to pollute the main Getting Started page labeled as "Install tools" when it was actually specific to macOS and the tutorials. See:
https://github.com/aptos-labs/aptos-core/blob/6f306e1337f4c3fde688d0176863b92f27c52369/developer-docs-site/docs/guides/getting-started.md#install-tools

Reply:
> we could probably make this generic or have tabs... I don't think there's much here that is mac specific except the brew commands

I have now gone through each tutorial, added a "Prerequisites" section, and moved references to only the packages needed there. See the latest commit.

Reply:
@davidiw see the discussion above here for more background.